java/get-elevation-at-point/README.md

```diff
@@ -28,9 +28,10 @@ Tap anywhere on the surface to get the elevation at that point.
 
 ## Additional information
 
-`getElevationAsync(surfacePoint)` retrieves the most accurate available elevation value at a given point which requires it to go to the server or local raster file and load the highest level of detail of data for the target location and return the elevation value.
+`getElevationAsync(surfacePoint)` retrieves the most accurate available elevation value at a given point. To do this, the method must go to the server or local raster file and load the highest level of detail of data for the target location and return the elevation value.
 
-If multiple elevation sources are present in the surface the top most visible elevation source with a valid elevation in the given location is used to determine the result.
+If multiple elevation sources are present in the surface, the top most visible elevation source with a valid elevation in the given location is used to determine the result.
 
-#### Tags
-MapViews, SceneViews and UI
+## Tags
+
+elevation, MapViews SceneViews and UI, surface
```

Comment:
remove the API from the tags.

Reply:
I think this is one of the category tags, which I've been told to leave in in some samples. @Rachael-E should this be removed?

Reply:
this is one I'm a little unsure of myself. I've seen it in a few others and have left them in since as Hudson says, it's a category tag. I think it's ok to leave it here, bearing in mind I think the tags are getting an overhaul/being done away with anyway at a later date.

Reply:
Ok, that sounds good to me.